src/test/java/de/rabea/game/GameSetupTest.java

```diff
@@ -0,0 +1,65 @@
+package de.rabea.game;
+
+import de.rabea.player.FakeComputerPlayer;
+import de.rabea.player.HumanPlayer;
+import de.rabea.player.PlayerFactory;
+import de.rabea.ui.FakeUserInterface;
+import de.rabea.ui.UserInterface;
+import org.junit.Before;
+import org.junit.Test;
+
+import static de.rabea.game.Mark.O;
+import static de.rabea.game.Mark.X;
+import static org.junit.Assert.assertTrue;
+
+public class GameSetupTest {
+    FakeUserInterface fakeUserInterface;
+
+    @Before
+    public void setup() {
+        fakeUserInterface = new FakeUserInterface();
+    }
+
+    @Test
+    public void setsUpFirstHumanVsHuman3x3Game() {
+        fakeUserInterface.provideConsoleInput("1", "3", "1", "7", "2", "4", "3", "n");
+        PlayerFactory playerFactory = new PlayerFactory(fakeUserInterface);
+        GameSetup gameSetup = new GameSetup(fakeUserInterface, playerFactory);
+        gameSetup.startApplication();
+        assertTrue(fakeUserInterface.greetUserWasCalled);
+        assertTrue(gameSetup.getPlayer() instanceof HumanPlayer);
+        assertTrue(gameSetup.getOpponent() instanceof HumanPlayer);
+    }
+
+    @Test
+    public void setsUpFirstHumanVsComputer3x3Game() {
+        fakeUserInterface.provideConsoleInput("2", "3", "1", "2", "3", "n");
```

Comment:
This test is pretty long and hard to read. Is there a simpler way to do the setup?